chromium/browser_type_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+ *
+ * xk6-browser - a browser automation extension for k6
+ * Copyright (C) 2021 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package chromium
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/grafana/xk6-browser/common"
+)
+
+func TestBrowserTypeFlags(t *testing.T) {
+	t.Parallel()
+
+	t.Run("devtools", func(t *testing.T) {
+		t.Parallel()
+
+		const devToolsFlag = "auto-open-devtools-for-tabs"
+
+		var bt BrowserType
+
+		flags := bt.flags(&common.LaunchOptions{
+			Devtools: false,
+		})
+		flagi, ok := flags[devToolsFlag]
+
+		if !ok {
+			t.Fatalf("%q is missing", devToolsFlag)
```

Comment:
We can rely on the `testify` API to do these checks, so this can be `require.True(t, ok)`. You can use `Truef()` if you want the message as well.

I typically use `require` instead of `assert` if the following code depends on the assertion passing.

Reply:
I'm not a fan 3rd party pkgs while we have `testing.T`. I'm not sure what value it brings. It's like using a pkg for skip writing `if err != nil`. It steals the expressiveness from code.

But.... Sure.

Reply:
It standardizes the way assertions are made, and makes tests more readable precisely by avoiding repetitive checks. I thought you were a fan of readability and abstractions? :)

Besides, it's a pretty widely adopted package, there's no harm in depending on it.

Reply:
`testing.T` is already standardized, and we have good conventions around it. Anybody who knows how to write idiomatic Go code already knows how to use it effectively.

I guess you misunderstood me before. I'm a fan of readability, but not useless abstractions. I believe in abstractions only if they don't take away any expressiveness from code. `if a != b { t.Errorf(...) }` is as expressive as `if err != nil`.